mne_neuron/dipole.py

```diff
@@ -36,8 +34,6 @@ def simulate_dipole(net, n_jobs=1):
     from neuron import h
     h.load_file("stdrun.hoc")
 
-    net = Network(net.params, n_jobs=n_jobs)
```

Comment:
hmm ... I don't understand how this fixes it. Could you explain the logic to me?

Since I used the same params which have the same random seed, shouldn't it give the same result? Unless the params are getting modified somehow inside the Network class -- which it shouldn't !

Reply:
Building the network twice is unnecessary, no? As for the why, it could be NEURON, or python. Regardless it's not idempotent.

Reply:
See comments in file

Reply:
@jasmainak here's a function that gets run by Network.__init__() that should only be called once:
https://github.com/jasmainak/mne-neuron/blob/master/mne_neuron/cell.py#L105

Other functions depend on NEURON for idempotency, which may or may not be safe:
network._create_coords_pyr
network._create_coords_basket
network._create_gid_dict
network.__gid_assign
network._create_all_src
network._record_spikes

Shall we just call Network() once? I would prefer in dipole.py somewhere to abstract NEURON network building from the user, at least until netpyne integration work begins. I recall your point in a previous PR about the API being more flexible and allowing the user to modify the network, but can we settle without that for now?

Reply:
@blakecaldwell sorry for the slow response rates. I have been busy with an MNE workshop the last few days and until around Friday.

I agree that the network building should be idempotent. That is, building the network and running the network should be somewhat independent of each other. If you have some time/efforts I would welcome changes to make this happen. 

Let's discuss on a separate issue how the API and/or abstraction would like. It's a bit weird to call Network with n_jobs because there is nothing in parallel when you build the network.

In any case, I can live without it for now if it helps you move forward. To merge this PR, could you address my [tiny comment below](https://github.com/jasmainak/mne-neuron/pull/47/files#r287550906)? Thanks a lot!